code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -469,14 +469,15 @@ Code:
 				menu += "<h4>Located Mops:</h4>"
 
 				var/ldat
-				for (var/obj/item/mop/M in world)
```

Comment:
Honestly I think the janitorial locator should just be ripped out. This code is terrible, someone can add GPS units to mops if they really care.

Reply:
Rip out both mops and carts? Mops are kinda useless since you can just print them but cart is unique.

Reply:
I'd say give the janitor a janicart pinpointer (even though cargo can order a new cart) but that should be done in a separate PR.